chain/client/src/stateless_validation/chunk_validator/orphan_witness_handling.rs

```diff
@@ -6,7 +6,6 @@
 //! arrives, all witnesses that were waiting for it can be processed.
 
 use crate::Client;
-use bytesize::ByteSize;
```

Comment:
It might be possible to remove the dependency on `bytesize` from `chain/client/Cargo.toml` now. AFAIR it was added only for this import.

Reply:
This is actually needed for config.rs using ByteSize starting with this PR.

Reply:
The config is in a different crate (`near-chain-configs`), so the dependencies in `chain/client/Cargo.toml` don't apply to it.
But I tried to remove this dependency and it fails because `bytesize` is needed by `state_witness_tracker.rs`, so it has to stay.